src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunnerForWorker.java

```diff
@@ -94,42 +95,42 @@ public void execute() {
           TextUICommandLine commandLine = new TextUICommandLine();
           FindBugs.processCommandLine(commandLine, args, findBugs2);
           findBugs2.execute();
+
+          StringBuilder message = new StringBuilder();
           if (findBugs2.getErrorCount() > 0) {
-            throw new GradleException(
-                "Verification failed: SpotBugs error found: "
-                    + findBugs2.getErrorCount()
-                    + ". "
-                    + buildMessageAboutReport());
-          } else if (findBugs2.getBugCount() > 0) {
-            throw new GradleException(
-                "Verification failed: SpotBugs violation found: "
-                    + findBugs2.getBugCount()
-                    + ". "
-                    + buildMessageAboutReport());
+            message.append(findBugs2.getErrorCount()).append(" SpotBugs errors were found.");
+          }
+          if (findBugs2.getBugCount() > 0) {
+            if (message.length() > 0) {
+              message.append(' ');
+            }
+            message.append(findBugs2.getBugCount()).append(" SpotBugs violations were found.");
+          }
+          if (message.length() > 0) {
+            String reportPath = findReportPath();
+            if (reportPath != null) {
+              message.append(" See the report at: ").append(Paths.get(reportPath).toUri());
+            }
+
+            GradleException e = new GradleException(message.toString());
+
+            if (params.getIgnoreFailures().getOrElse(Boolean.FALSE).booleanValue()) {
```

Comment:
Sorry, but shouldn't there be a logical not here?

Reply:
Not sure I follow what you are asking for, if `ignoreFailures` is true then warning messages are logged, otherwise the exception is thrown.

Reply:
you're right. I was thinking "ignore failures, but wait we are actually doing stuff instead of ignoring it!". You are correct. Everything looks good to me!